content/en/docs/configuration/ca.md

```diff
@@ -55,7 +55,15 @@ spec:
   ca:
     secretName: ca-key-pair
 ```
-
+Optionally, you can specify (CRL)[https://en.wikipedia.org/wiki/Certificate_revocation_list] Distribution Points. An array of `Strings` each of which identifies the location of the CRL from which the revocation of this certificate can be checked:
```

Comment:
Any particular reason we backtick Strings? I think just `strings` (with not backticks) will do in this case

Reply:
done